app/components/registrations_and_follow_ups_component.html.erb

```diff
@@ -39,7 +39,7 @@
         </tr>
         <tr class="sorts" data-sort-method="thead">
           <th class="row-label sort-label sort-label-small ta-center" data-sort-default>
-            Facilities
+            <%= region.child_region_type.capitalize %>
```

Comment:
Will this show the pluralized word?

Reply:
I left it singular to match the behavior on the `Overview` report